src/server/qgsserverfilter.h

```diff
@@ -59,16 +59,62 @@ class SERVER_EXPORT QgsServerFilter
 
     /**
      * Method called when the QgsRequestHandler is ready and populated with
-    * parameters, just before entering the main switch for core services.
-    */
-    virtual void requestReady();
+     * parameters, just before entering the main switch for core services.
+     *
+     * This method is considered as deprecated and \see onRequestReady should
+     * be used instead.
+     *
+     * \deprecated Will be removed in QGIS 4.0
+     */
+    Q_DECL_DEPRECATED virtual void requestReady() SIP_DEPRECATED;
+
+    /**
+     * Method called when the QgsRequestHandler processing has done and
+     * the response is ready, just afte the main switch for core services
```

Comment:
```suggestion
     * the response is ready, just after the main switch for core services
```

Reply:
Fixed

Also modified `/since` annotations to `3.24`